CodeEditModules/Modules/AppPreferences/src/Sections/SourceControlPreferences/SourceControlGitView.swift

```diff
@@ -27,17 +27,17 @@ struct SourceControlGitView: View {
             }
 
             PreferencesSection("Ignored Files", hideLabels: false) {
-                if prefs.preferences.sourceControl.git.ignoredFiles.isEmpty {
-                    Text("No Ignored Files")
-                        .background(Color(NSColor.textBackgroundColor))
-                } else {
-                    List($prefs.preferences.sourceControl.git.ignoredFiles,
-                         selection: $ignoredFileSelection) { ignoredFile in
-                        IgnoredFileView(ignoredFile: ignoredFile)
+                List($prefs.preferences.sourceControl.git.ignoredFiles,
```

Comment:
Add a 1pt border around the list so it better separates from the rest.

<img width="1016" alt="Screen Shot 2022-04-14 at 00 22 17" src="https://user-images.githubusercontent.com/9460130/163280036-171c1d66-c4b3-4d57-ab8c-4970d3d1a987.png">

Reply:
Just add a padding of 1 and a background with NSColor.separatorColor

See how it is done on the main container of the view